### PR TITLE
 Added support for non Administrator CLI user

### PR DESCRIPTION
**Pull request by leader80**
In case of non-admin user instead of symlinkSync will just writeFileSync.

### CODE REVIEW

**Comment by leader80** [comment]
I signed it!

**Comment by leader80** [comment]
it fixes problem:  #370

**Comment by leader80** [comment]
Fixes:  ng serve => Error: EPERM: operation not permitted, symlink #641

**Comment by reflexdemon** [comment]
When will this be merged?

**Comment by filipesilva** [comment]
@leader80 thanks so much! I'm looking at this PR right now, and should have it merged soon.

File: lib/broccoli/broccoli-typescript.js [hunk 1]
```diff
@@ -78,7 +78,12 @@ class BroccoliTypeScriptCompiler extends Plugin {
         this._fileRegistry[tsFilePath].outputs.forEach(absoluteFilePath => {
           const outputFilePath = absoluteFilePath.replace(this.cachePath, this.outputPath);
           fse.mkdirsSync(path.dirname(outputFilePath));
-          fs.symlinkSync(absoluteFilePath, outputFilePath);
+          try {
+            fs.symlinkSync(absoluteFilePath, outputFilePath);
+          } catch (e) {
+            const conentStr = fs.readFileSync(absoluteFilePath);
```
Comment (mernen) [inline lib/broccoli/broccoli-typescript.js]: Shouldn't it be con**t**entStr?

Comment (filipesilva) [inline lib/broccoli/broccoli-typescript.js]: Good catch, will fix it in another PR.

Comment (filipesilva) [inline lib/broccoli/broccoli-typescript.js]: Fixed in https://github.com/angular/angular-cli/pull/981

**Comment by filipesilva** [comment]
These changes should be in the upcoming `beta.6`.

**Pull request closed without merging**